packages/app/hooks/use-connect-apple-music/utils.ts

```diff
@@ -0,0 +1,6 @@
+export const developerToken =
```

Comment:
oh this needs to be moved into Next's API or our backend. I think backend is better. Will ping Max.